src/cmd/health.js

```diff
@@ -139,20 +158,9 @@ class NSFSHealth {
         let bucket_details;
         let account_details;
         const response_code = endpoint_state ? endpoint_state.response.response_code : 'NOT_RUNNING';
-        const rsyslog = await this.get_service_state(RSYSLOG_SERVICE);
-        let service_health = 'OK';
-        let syslog_ng_health = 'OK';
-        if (this.check_syslog_ng) {
-            syslog_ng = await this.get_service_state(SYSLOG_NG_SERVICE);
-            if (syslog_ng.service_status !== 'active' || syslog_ng.pid === '0') {
-                syslog_ng_health = 'NOTOK';
-            }
-        }
-        if (syslog_ng_health === 'NOTOK' || service_status !== 'active' || pid === '0' || response_code !== 'RUNNING' ||
-            rsyslog.service_status !== 'active' || rsyslog.pid === '0') {
-            service_health = 'NOTOK';
-        }
-        const error_code = await this.get_error_code(service_status, pid, rsyslog.service_status, response_code);
+        const service_health = service_status !== 'active' || pid === '0' || response_code !== 'RUNNING' ? 'NOTOK' : 'OK';
```

Comment:
@shirady I already did this in the refactoring PR but since we don't want to backport refactoring I splitted the PR to this (mostly fixes) and a large refactoring PR

Reply:
to be continued...